### DISCUSSION TITLE
Project admin not active

### DISCUSSION THREAD

**Discussion by omkarkhopade31** [sanjay-kv/Open-source-Practice#988] [Q&A] [answered]
what to do if the project admin is not active

**Comment by PRAYAG0908** [comment, marked as answer]
add that repository url in https://github.com/GSSoC24/Contributor/discussions/94 gssoc will flag that admin

**Comment by Manas200426** [comment]
add that repository url in GSSoC24/Contributor#94 gssoc will flag that admin

**Comment by omkarkhopade31** [comment]
> what to do if the project admin is not active
